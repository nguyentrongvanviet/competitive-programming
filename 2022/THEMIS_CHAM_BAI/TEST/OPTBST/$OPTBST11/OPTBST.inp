1713
892511301 914385965 792360362 574546424 263679794 252030505 855909422 783761031 640830479 175835058 44443493 654987855 862607736 291902270 174550740 597839325 83343733 193265353 180083691 282347259 921802759 471674896 216791273 847840808 22518318 406018512 39390612 342187571 230268266 591739271 773961505 791878452 626676178 527731769 835041978 243045892 6791992 632746044 645217681 41019047 378945291 896698989 885712653 521224961 672774213 799531359 147369777 532369923 591273970 622927150 126848579 21002087 738955616 396157551 260913740 880426596 324572379 500221440 246241083 697724365 600930346 751553491 170530555 764546901 775977233 621148846 127330114 910389972 305845950 370790073 473098414 189516013 736692576 131887953 728009752 866395990 651026278 258015485 352554180 260134332 422242720 618659885 401758536 130727445 507833765 905865603 590856924 48520507 647115160 581064390 43809402 511864201 103660737 828190854 889033477 211180448 893803844 590921856 555990894 68543680 817890130 424753851 260040495 80972761 642567586 434411756 212386667 676008919 458034207 443190852 675046549 309044263 90930680 200978431 7931907 766999569 569362375 939426333 756464336 750401160 347708210 12730923 162013282 220053069 651585854 736202322 83790081 720699160 906466766 421206653 471718929 261354930 139509203 679977305 77133584 195032668 830769345 646663684 597269843 453768288 649669707 272034208 824380961 776481046 256671775 247766525 261189795 181246073 364729851 545619378 779458579 126578644 85325371 63296251 299385140 21678304 169442875 729318226 546036654 210690240 358702637 641682155 264540363 669760464 357732533 125780107 569422490 916235626 286779871 508232675 658699049 612856489 423292877 331242432 21787641 533937190 148743158 558051332 13139883 60603158 357807299 142060544 337718539 149803557 832610377 143122997 294664335 521361718 833581790 731071751 641687866 885794194 14128046 713998177 263708209 570694817 572476743 181023688 565997343 542964822 807394941 254420456 845956994 641708331 325962286 234567606 202873429 757911426 86903831 404839231 914265168 869553073 884523641 195987246 117399407 184053702 296251626 548800623 880700901 624291349 937754197 261197041 494138467 17033677 82652175 251972760 267055498 436018051 322406767 436046932 561200941 744946882 598041516 27246408 664170626 310888290 272430182 195415920 791521938 572790015 932247752 709180353 414676625 841346941 813394256 851766034 529266081 847325771 834335123 761452768 715247358 338818617 63914185 237652829 853421871 188659871 870913128 13882904 598557513 709199234 651315087 742394845 432826158 28222954 686347618 5605478 255181014 680911603 442105719 128562742 227524209 502752418 770999954 762711009 549842392 379571743 236527801 646557450 901099524 265989878 503221373 172427110 922314154 902854345 717222579 508464182 8821958 585019429 464490309 389400230 407724595 374200437 166561572 109061482 837377715 788127549 61373152 496549981 822224923 400278956 306857450 126228569 215703033 544727610 590755945 605333602 780647079 637000641 356551049 162019566 605816593 472295476 202352751 161833956 397591553 768695914 843592401 155669714 120306445 490021361 406053475 474193139 269145762 11917261 468917399 455716197 667233637 936870927 177104562 13799461 365059273 186269871 521355480 779388996 556100379 459890546 547862388 266578099 212536474 827286439 538864579 495245065 938397392 330158694 199855886 244037475 751076159 481619165 784788132 240227128 375329210 893992820 441961900 66122167 920772471 244542244 145810149 243565545 468925597 666435417 651671672 927242241 914908990 339985272 876541061 477180077 193962715 823659947 468897409 903190987 472157855 26044824 441598044 737958536 533448203 350714552 107697821 924783703 417663607 71717013 509774929 212658406 613401291 395970862 904875663 582569010 784154426 57688034 71332609 809067477 563998576 649402803 319165186 923300036 366144909 197355084 379748638 374781113 711830894 754727755 921301664 23011940 571236072 561797603 531723101 466651331 753298428 87486493 388769359 732651728 784616809 437412036 100621875 374471668 200948722 142198984 697084218 722681678 642234081 703128229 699735479 458676671 297739792 422219007 823849304 7807743 390008442 137948604 668726207 369375098 687950628 276845703 155836506 548178126 559329898 365740578 47498838 204530882 378111779 797157508 320239755 732445647 477641307 431827639 898657777 543582091 721728703 641321091 723743161 446626375 117921893 697855819 434244948 591990064 253569178 872987857 848001762 567678305 441794284 468341275 622422992 297450582 416450366 73967357 709837542 931326140 1543381 260922968 191495283 871221086 14417882 339365088 106143808 499893102 836993512 674692548 613143462 508697370 130786139 664043989 789213051 687984054 695038417 510761950 157060585 895734840 665536027 184893513 895016130 687438288 795921225 437491706 473827181 916739526 562017154 431172632 853007916 812995881 739549318 123156804 340326374 472022943 881640751 87021198 212835935 595991750 13951812 97983532 511085046 500637704 779533043 933092307 263668432 643324915 510477656 377810706 189285679 890776042 292453995 171180288 840858250 838035725 314836904 835628592 483916725 704548988 659850822 582660602 188252032 69111767 12695486 661463663 572097841 773650284 378015347 776547177 465246211 861057329 498733160 481745936 101171593 333562673 497323483 566834434 772266340 499597290 109763487 181774612 851989660 663553916 333015454 462613145 132986331 299879042 758046359 426007918 652661154 288665975 302500196 695102743 372688349 605957598 613757316 19755053 328020839 895934349 392252674 887473045 842614148 677228753 670380419 53866614 140393789 576072325 515528697 545461522 780947308 813374065 903674199 93652979 581297448 394149285 197446130 552309139 263479947 923455849 332108076 517023519 902387821 356968493 734104801 705715868 640013428 843277334 26868832 757519674 210232169 756369577 450468177 302571277 932631415 831580822 385454674 216476213 177327433 405234692 508710968 520947285 455714425 18437417 925605703 260073561 349300030 755579860 889475939 321845946 907414302 413793351 285360190 847498503 2180028 829747488 404882979 777604448 79942416 32073772 769110146 695601837 492491541 704426890 781697826 671427184 289132580 454070947 537639561 831472468 494168871 440075823 854158046 795675737 54959237 167168987 82081994 115948670 290132266 750095498 341553563 687907893 386296186 733107138 208390093 236064766 808281765 130126350 92077328 556708750 833488942 582714728 80192909 119447938 250729487 162091395 858508780 129914010 780103285 38856849 77973267 920993101 411712723 417885981 445203655 59903915 614394477 352999790 852699563 472653392 907428779 846391039 933710887 105446204 331545779 177131292 935050599 726990034 294504155 145604097 289563974 477111678 823189141 880577736 881728686 397843411 217793578 733764165 836557207 559430973 70467897 652497983 621841289 653644850 373651311 238595152 320239624 703614553 184182289 608569261 672296548 755258983 905881661 437856901 450259198 927439460 688383642 929108490 789187068 925008132 757005577 222955529 802843885 271803579 745985690 783505571 566283695 915562209 149121826 118643980 280643410 844749275 629289858 880615442 372461568 100215452 694430579 740881504 834117647 6909589 246805967 700008903 857269043 382236443 817042839 15984914 313256340 69627804 527882691 575093623 483755829 224499690 844551552 124995019 802906587 353317865 381219792 446568141 37355977 782954989 569100311 419556825 823909262 675883065 479676192 227364384 804061896 497405365 498430232 192646767 654774160 261628348 402443367 827514516 801124516 395710398 298007706 89823234 708637586 392418847 632080594 450712744 293588200 527505647 27847733 727714077 126441770 59569385 797243878 904239985 408931632 817002490 173207143 621455436 364484830 507641156 590436870 768429136 370645425 176165910 380203920 491065447 134492096 244084862 860245839 515376564 362637696 544421667 389999890 475815973 635995880 664800558 470953098 621156305 474220195 258961026 572555945 261013453 303293641 155571246 133042360 937962623 905737227 938190199 9889327 889382373 35514615 575784886 393005131 658764156 348803574 305854970 47670293 506545856 396515409 455174724 259399006 456765422 526789769 668455421 131484783 169694678 552317944 175518425 342185173 784969206 320568264 170429325 422202119 353366622 577011777 292625054 673137573 434040967 596794770 138325968 443608676 622033793 103567491 856459200 140755798 17914635 210088570 360354973 429668294 470635516 849735545 637122275 320103721 381662442 682085874 593737482 91777271 583031543 693486945 140743804 370342528 803827372 716092449 936181313 588610219 101152412 871431700 478978580 305532449 620377434 867529622 312192075 96904983 193250015 635519941 278340362 26458251 877953511 38130883 364727348 434427865 365050202 772262961 564566997 180681297 631638396 272255864 234092857 324719301 495157131 696760473 86442755 316304915 880010831 528582596 115648341 3215666 700824451 31028248 760894359 540066941 48643038 596706577 438036515 123274016 177019889 386920642 675458371 490945093 506559053 444336854 602093717 555861364 326230738 42663222 339781511 877133095 537062096 84192554 201834377 269032839 731027307 141678655 158455208 344668409 923876081 739088537 191006104 810211865 560130549 732997371 331180095 236628760 771464998 603279210 631856469 173972883 785225978 907872323 238311409 424552831 904533116 563419417 102885772 235246295 908974653 474334533 868507653 286106237 26568505 913423613 689004241 109875790 272148452 691442343 193152202 408121455 824378415 323625903 793525411 598791292 265846877 157098611 144572717 639430210 366618591 624064104 418132580 243678689 25631838 568131114 734140232 331559982 918979949 839146344 489389346 505805779 903845087 139831984 757586457 816329595 908978345 424344261 261022763 261266857 929121270 382498 630231586 133026778 506291564 317035732 63147073 108592535 866385449 927118289 484029939 517697941 234095405 598217752 596694746 553557382 1938539 353969401 758615141 131802386 370194487 849227646 12650787 509903567 770449258 421334824 634219393 218351596 119559041 351296931 358828393 685223865 270368509 137260651 430084655 128954004 29982999 657466433 37916706 768606279 62300841 277645857 226128813 552944748 507604064 337584688 258870100 854440078 922288950 269995248 185352049 15030675 822916535 164008841 500895187 750132428 351718360 219261331 590779968 35080757 740404055 858172392 663558716 185185701 113191773 384074252 300432927 612758821 752426691 269396964 309458095 191019643 664842434 483903136 359911647 344729772 627437306 872751398 328213350 901148465 323355530 591413892 803540179 665203987 180165403 618688464 140197431 433312038 580551697 623854007 702199039 652978167 163206293 917196267 646361369 167011136 622528476 140456053 497559770 75517407 453011550 250760244 691607593 915948732 826366584 715183560 136263086 367609437 453068120 189834914 158684955 346741966 615414866 295615282 812895135 879608206 570510853 657589577 113120215 653753767 916503317 212908846 531881230 534988493 274838696 806285711 312882777 903500601 501461714 531636099 140253151 645926586 832204333 316294053 125422588 249521847 621553303 191064875 82330806 439319191 594402644 802666821 753573586 14180813 724716648 190254939 94243705 922518496 421733102 52749226 522238681 312229578 489443618 360674448 368801759 151429967 322957613 546129910 200009003 156404973 803139282 133007316 727147575 520857412 715109297 72974063 100390583 343139709 344910749 331873000 477307789 787952329 574449587 711935160 923476832 78835368 126230742 466498436 716417227 291908722 758968668 661906508 698121706 563813967 20155433 529162735 907768419 937771150 133312633 654188556 57775218 615801154 702054099 157289693 558952153 350333343 650015793 442805150 936911312 667215568 815513042 62258742 675635543 406953166 758697390 162354566 484690278 627018860 342185690 853392665 534754199 919637934 487551417 342686604 850752749 875395847 106935643 598691350 362918989 347628601 642365350 17234272 634234996 273101390 220907778 128009663 693414974 294804104 429953771 347503169 102732447 345385505 724471628 351263763 156488158 722943068 748696501 762976434 345239362 687282642 457610653 239233305 386244955 639536380 852928558 348010343 125513571 749755469 21720246 421659074 934141657 805333916 628203330 656608098 422952574 607404653 300767250 861838000 653273161 378763803 325478753 498738077 329771445 232704998 80191758 615461613 26139241 931753002 11135528 500157649 899533704 442895980 517350006 188839851 791424962 493738717 471985450 13565476 266921325 165734462 505451638 536417004 573299432 653846533 674315420 556162436 794003247 144824740 927674379 383888187 707099037 339763581 812664189 828649644 371211917 623186620 683770623 375155760 874492797 496687486 889979253 232872450 252583503 759847917 674739577 401869621 71994994 782803782 480913039 678948413 139574153 116274369 473664164 634171115 641266036 908867231 729340891 68076437 796096497 768421466 512224353 108995392 13988819 168271217 89861892 167705171 695873183 84482345 535997241 921706632 830519466 8631782 391621478 225797502 918799157 113215563 735199477 410446742 641907661 783851431 934225476 420782058 136237313 902473151 380967300 654914608 623161877 855618049 173620186 370735467 327245385 133623950 371000621 529124817 525297195 441649344 519236071 201499326 931936780 39576169 37487529 406626185 332039831 70516875 54099211 131175480 483636481 402744371 22343428 695019514 113279701 115560509 407861359 384006077 902092859 207610690 606862384 630636201 404788672 42385999 468945119 179740185 337689510 465643551 419119945 653801195 252213484 99650664 488633112 745040545 675245754 707142825 89471800 894082846 648959841 925068562 274368747 135679436 893210806 78596028 865935441 412677021 341692000 295199807 609186632 822254280 515812463 872321103 494812413 842162833 367368799 185004003 126027680 864403875 671617190 645789652 261670212 677411922 133353103 477856376 704181788 457355003 511696968 911362708 617947426 418046541 302287280 926532414 662010296 302112140 72003684 543230513 364144750 260187998 190872286 434485098 68391478 10219199 191770284 150842222 479468643 551471039 489974034 562863638 289064333 802754816 250650352 785282079 746368795 901389693 609234842 803799261 213674665 636082359 175455085 137941386 481225018 193907638 114989942 676549301 624437212 701925155 523671100 379467324 935342022 922804069 889282570 4550954 433155643 422249474 784047645 419896354 534876193 613509625 350816547 392108039 748231536 515168595 435742930 329342171 179508247 728670948 326097766 428238256 460672274 571285809 495339842 617099545 598047593 850063316 74074465 404872198 356195304 832032271 524027793 144172885 688448957 522668391 27713078 537586393 229393436 389884752 940418386 747250105 891223177 182990890 447286674 760799011 785991905 391247038 712389391 114637282 713142370 236126965 699315816 126114649 447992973 237391500 760775548 651309936 486003386 716278385 730682950 639268179 738524506 926423459 88261732 47446064 364532757 328546007 677929161 476954553 568886725 545362913 744980891 578584827 635807250 730395376 536622752 305209792 434848111 527873541 319668702 908705964 125734265 702490599 712541910 699374376 880547869 42833103 170676302 142117429 787924478 358003476 2311246 755302338 863485618 325147570 923658458 231256127 95201160 13026938 452402691 775030760 251611518 867905816 359435147 231802599 674612549 523901732 382612205 10150607 162063661 355457369 154777490 340287664 810959075 702093742 50133132 607692376 438855020 641850251 642633263 667954549 860689126 349377409 845555997 319474421 339449757 794525267 174814713 805737676 718630873 557733695 504746054 770620726 696786924 492761998 299380750 717379849 406291836 536610479 789207005 265715049 350777348 745107208 791099855 22130891 492076528 131633568 862500572 839710380 181303124 361551011 545149183 457334710 730138470 48550516 796792580 650694572 330429474 513198102 524538524 856394303 585944498 87734762 48695956 62819861 55094519 933602573 535164204 56820065 352590608 931274110 883664991 18559066 508945080 580725772 25252043 320333243 875447827 830219048 589656853 156716001 294531555 282259253 291920931 397736918 449668763 658756054 673744065 454531770 566099587 574684387 515622591 351799745 680699771 864567488 460620753 835167067 505688255 383522147 669943037 274052119 98056957 836790157 127915999 861348283 800282131 696151484 705785130 456771610 503240436 900920615 105275166 155064367 815770363 858612959 365252421 500041337 238882383 613122659 845680041 393771341 620762711 725372262 240568824 652934296 27127986 500787689 715747624 49860320 939270879 697658036 577427453 581935000 875258092 170345331 632488025 325197103 240557356 26529627 233582190 140740244 875636502 762976342 717590920 707209730 328963039 703099755 418823147 470543559 763655950 857313222 708354378 97741992 