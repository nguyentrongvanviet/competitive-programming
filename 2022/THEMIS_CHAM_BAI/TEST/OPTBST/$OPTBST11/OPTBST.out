7600235259865